library/src/main/java/io/ona/kujaku/comparisons/Comparison.java

```diff
@@ -0,0 +1,9 @@
+package io.ona.kujaku.comparisons;
+
+public abstract class Comparison {
```

Comment:
I think this could be an interface instead?

There is no default functionality (concrete methods).

Reply:
The type's supported by the `Comparison` class which is declared as a variable and used later by child classes